docs/UsersGuide/source/RunSRW.rst

```diff
@@ -0,0 +1,1241 @@
+
+If users have access to NOAA :term:`HPSS` but have not pre-staged the data, they can simply set the ``RUN_TASK_GET_OBS_*`` tasks to true, and the machine will attempt to download the appropriate data from NOAA HPSS. In this case, the ``*_OBS_DIR`` paths must be set to the location where users want the downloaded data to reside. 
+
+Users who do not have access to NOAA HPSS and do not have the data on their system will need to download :term:`CCPA`, :term:`MRMS`, and :term:`NDAS` data manually from collections of publicly available data, such as the ones listed `here <https://dtcenter.org/nwp-containers-online-tutorial/publicly-available-data-sets>`__. 
+
+Next, the verification tasks must be turned on according to the user's needs. Users should add some or all of the following tasks to ``config.yaml``, depending on the verification procedure(s) they have in mind:
+
+.. code-block:: console
+
+   workflow_switches:
+      RUN_TASK_VX_GRIDSTAT: true
+      RUN_TASK_VX_POINTSTAT: true
+      RUN_TASK_VX_ENSGRID: true
+      RUN_TASK_VX_ENSPOINT: true
+
+These tasks are independent, so users may set some values to true and others to false depending on the needs of their experiment. Note that the ENSGRID and ENSPOINT tasks apply only to ensemble model verification. Additional verification tasks appear in :numref:`Table %s <VXWorkflowTasksTable>`. More details on all of the parameters in this section are available in :numref:`Section %s <VXTasks>`. 
+
+.. _GenerateWorkflow: 
+
+Generate the Regional Workflow
+-------------------------------------------
+
+Run the following command from the ``ufs-srweather-app/ush`` directory to generate the workflow:
+
+.. code-block:: console
+
+   ./generate_FV3LAM_wflow.py
+
+The last line of output from this script, starting with ``*/1 * * * *`` or ``*/3 * * * *``, can be saved and :ref:`used later <Automate>` to automatically run portions of the workflow if users have the Rocoto workflow manager installed on their system. 
+
+This workflow generation script creates an experiment directory and populates it with all the data needed to run through the workflow. The flowchart in :numref:`Figure %s <WorkflowGeneration>` describes the experiment generation process. First, ``generate_FV3LAM_wflow.py`` runs the ``setup.py`` script to set the configuration parameters. Second, it copies the time-independent (fix) files and other necessary data input files from their location in the ufs-weather-model directory to the experiment directory (``$EXPTDIR``). Third, it copies the Weather Model executable (``ufs_model``) from the ``exec`` directory to ``$EXPTDIR`` and creates the input namelist file ``input.nml`` based on the ``input.nml.FV3`` file in the ``parm`` directory. Lastly, it creates the workflow XML file ``FV3LAM_wflow.xml`` that is executed when running the experiment with the Rocoto workflow manager.
+
+.. COMMENT: ufs-weather-model directory doesn't exist... What was intended here?
```

Comment:
That would make sense... but I'm confused why the fix files would be coming from there at all. I thought they were pre-staged on L1 platforms, and normally the directory path doesn't have `ufs-weather-model` in it. @JeffBeck-NOAA @gsketefian Do either of you know?

Reply:
@gspetro-NOAA @JeffBeck-NOAA  @gsketefian Very good point.  Looking at the scripting, it appears as though the fixed files are being copied from the system (FIXgsm) into the experiment directories (FIXam), not directly from `ufs-weather-model`.  It would appear that users not on a Tier-1 machine need to pre-stage the fixed files before being able to run the SRW.  If this is the case, where can one go about finding these fixed files?

Reply:
The fix files used to come from specific locations on disk that were maintained by EMC for ufs-weather-model use.  We're no longer doing that, and are instead sourcing our own copies of the fixed files from SRW App-specific directory locations on each platform.  @mkavulich lead that effort.  @MichaelLueken, you're also correct that users not on the main NOAA HPC machines (and a few others) need to manually stage their own fixed files.  They're available on an AWS site for download (see [here](https://ufs-srweather-app.readthedocs.io/en/develop/InputOutputFiles.html#static-files) for related documentation).